### PR TITLE
{math}[foss/2023a] ELPA v2023.05.001

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
tcn30.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/b82f7a9a429b4647670b1ff0d9af4d4d for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/ea0029863a0b77f426bbf9af770100e6 for a full test report.

**Comment by casparvl** [comment]
My failure only happened in a job environment, so it has something to do with that environment. I think we can safely ignore that, as it builds correctly if I just ssh into the node.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ generoso
EB_ARGS="--ignore-locks"

File: easybuild/easyconfigs/e/ELPA/ELPA-2023.05.001_fix_AVX512_support.patch [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# What: Fixes support for AVX512 on AMD CPUs
```
Comment (casparvl) [inline easybuild/easyconfigs/e/ELPA/ELPA-2023.05.001_fix_AVX512_support.patch:1]: Can you elaborate? Is there an upstream issue? What is the error you see when you don't have this? (and is that at build or runtime?)

**Review by casparvl — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/e/ELPA/ELPA-2023.05.001_fix_AVX512_support.patch (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/e/ELPA/ELPA-2023.05.001_fix_AVX512_support.patch]: done

**Comment by maxim-masterov** [comment]
I found an open repo for ELPA (mirror?) and opened an issue regarding the AVX512 support: https://github.com/marekandreas/elpa/issues/39

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/f88557a572346b87e3c517303aaea480 for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**